### PR TITLE
Expose training hyperparameters to allow easier hyperparameter tuning

### PR DESCRIPTION
**Pull request by dizcology**
This PR exposes the following learning hyperparameters in flags:

- base learning rate
- momentum
- weight decay

### CODE REVIEW

**Comment by alope107** [comment]
Thanks for your interest and PR! Unfortunately, as our README states, this repository is a public mirror, pull requests will not be accepted. Please file an issue if you have a feature or bug request.
